azure-pipelines/templates/1espackage.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - task: Npm@1
+    displayName: "\U0001F449 Package"

```

Comment:
This is the 👉 emoji. It makes it much easier to visually separate what is from our template vs. what is from 1ES templates.